pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://webdev.dartlang.org
 
 environment:
-  sdk: '>=2.0.0 <3.0.0'
```

Comment:
This pubspec is for the site's tooling. Leaving it at 2.0.0 should be ok.

Reply:
Unless, of course, you know that some of the packages _require_ 2.1.0.

Reply:
Ah, I wondered about that. Thought I'd have Travis test this. :)

Reply:
Travis won't complain about having a tighter bound. It'll just prevent us from running the local tools on an SDK below 2.1.0.